corehq/apps/accounting/migrations/0055_linked_projects.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations
+from django.core.management import call_command
+from corehq.util.django_migrations import skip_on_fresh_install
+
+from corehq.privileges import LINKED_PROJECTS
+
+
+@skip_on_fresh_install
+def _grandfather_basic_privs(apps, schema_editor):
+    call_command('cchq_prbac_bootstrap')
+    call_command(
+        'cchq_prbac_grandfather_privs',
+        LINKED_PROJECTS,
+        skip_edition='Paused,Community,Standard,Pro,Advanced',
```

Comment:
Are you certain there isn't anyone on a non-enterprise subscription who currently has the flag on?

Reply:
There are non-enterprise subscriptions that use this feature, but we will handle that separately either by migrating over to custom software plans with this privilege included, or keeping the feature flag available for their use.